src/d3d9/d3d9_common_texture.cpp

```diff
@@ -156,6 +156,16 @@ namespace dxvk {
 
     if (pDesc->Width == 0 || pDesc->Height == 0 || pDesc->Depth == 0)
       return D3DERR_INVALIDCALL;
+
+    // Native drivers won't allow the creation of DXT format
+    // textures that aren't aligned to block dimensions.
```

Comment:
does this requirement also pertain to smaller mips or only the top level, i.e. is something like 12x8 with two, three, four mips allowed respectively?

Reply:
To be fair, I haven't checked, but the Wine tests at least only validate the top level.